client/ayon_houdini/plugins/workfile_build/load_placeholder.py

```diff
@@ -42,9 +71,132 @@ def collect_placeholders(self):
         load_placeholders = self.collect_scene_placeholders()
 
         for node in load_placeholders:
-            placeholder_data = read(node)
+            placeholder_data = self._read(node)
             output.append(
                 LoadPlaceholderItem(node.path(), placeholder_data, self)
             )
 
         return output
+
+    # Take control of loaded repre data
+    def load_succeed(self, placeholder, container):
+        # Move the container to the placeholder and transfer connections
+        placeholder_node = hou.node(placeholder.scene_identifier)
+        target_context = placeholder_node.parent()
+
+        # If placeholder node is an object merge then do not move the container
+        # node, but instead set the object merge to point to the container node
+        if placeholder_node.type().name() == "object_merge":
```

Comment:
This is one possible suggestion. to use object merge if placeholder is in sops.
```suggestion
        if placeholder_node.type().category().name() == "Sop":
```